### PR TITLE
Stabilize AutocompleteProposalJavaDocTest and DeleteProjectsTest

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Stabilize AutocompleteProposalJavaDocTest and DeleteProjectsTest

### What issues does this PR fix or reference?
Issue https://github.com/eclipse/che/issues/6961

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 1]
```diff
@@ -53,13 +54,16 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** @author Musienko Maxim */
 @Singleton
 public class CodenvyEditor {
 
   public static final String CLOSE_ALL_TABS = "gwt-debug-contextMenu/closeAllEditors";
   public static final String VCS_RULER = "//div[@class='ruler vcs']/div";
+  public static final Logger LOG = LoggerFactory.getLogger(CodenvyEditor.class);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: Use static import to improve readability.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java [hunk 2]
```diff
@@ -93,8 +90,7 @@ public void shouldDisplayJavaDocOfClassMethod()
     editor.selectAutocompleteProposal("concat(String part1, String part2, char divider) : String");
 
     // then
-    verifyJavaDoc(
-        editor.getAutocompleteProposalJavaDocHtml(),
+    editor.waitContextMenuJavaDocText(
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java]: Inline, please.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java [hunk 3]
```diff
@@ -173,8 +166,7 @@ public void shouldDisplayJavaDocOfJreClass() throws IOException {
     editor.selectAutocompleteProposal("hashCode() : int");
 
     // then
-    verifyJavaDoc(
-        editor.getAutocompleteProposalJavaDocHtml(),
+    editor.waitContextMenuJavaDocText(
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java]: Inline, please.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java [hunk 4]
```diff
@@ -208,17 +200,10 @@ public void shouldWorkAroundAbsentSourcesOfExternalLib() throws IOException {
     editor.selectAutocompleteProposal("info(String msg) : void");
 
     // then
-    verifyJavaDoc(
-        editor.getAutocompleteProposalJavaDocHtml(),
+    editor.waitContextMenuJavaDocText(
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java]: Inline, please.

**Review by tolusha — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 5]
```diff
@@ -1759,12 +1763,37 @@ public void clickOnItemInContextMenu(String item) {
...
   public String getAutocompleteProposalJavaDocHtml() throws IOException {
+    waitJavaDocPopupSrcAttributeIsNotEmpty();
+    return getJavaDocPopupText();
+  }
+
+  public void waitContextMenuJavaDocText(String expectedText) {
+    waitJavaDocPopupSrcAttributeIsNotEmpty();
+
+    loadPageDriverWait.until(
+        new ExpectedCondition<Boolean>() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: Please, rewrite with lambda.

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**